### PR TITLE
add tests for win11 cpu status

### PR DESCRIPTION
**Pull request by PrzemyslawBanasiak13**
Signed-off-by: pbanasiak <przemyslaw.banasiak@3mdeb.com>

### CODE REVIEW

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-compatibility/31T-cpu-status.md [hunk 1]
```diff
@@ -89,6 +113,56 @@ LEVEL4_CACHE_ASSOC                 0
...
+
+1. Proceed with the [Common](#common) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Wait for the `OPERATING_SYSTEM` to boot.
+1. Run PowerShell as an administrator and execute command:
+
+```powershell
```
[the remark this answers is not in the document]
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/31T-cpu-status.md]: Done

File: docs/unified-test-documentation/dasharo-compatibility/31T-cpu-status.md [hunk 2]
```diff
@@ -110,9 +184,9 @@ Check whether the DUT has multiple CPU support.
 1. Wait for the `OPERATING_SYSTEM` to boot.
 1. Execute below command in terminal:
 
-    ```bash
-    lscpu
-    ```
+```bash
```
[the remark this answers is not in the document]
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/31T-cpu-status.md]: Done

File: docs/unified-test-documentation/dasharo-compatibility/31T-cpu-status.md [hunk 3]
```diff
@@ -154,9 +265,9 @@ Check whether the DUT has multi-core support.
...
```
[the remark this answers is not in the document]
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/31T-cpu-status.md]: Done

File: docs/unified-test-documentation/dasharo-compatibility/31T-cpu-status.md [hunk 4]
```diff
@@ -176,3 +287,56 @@ Core(s) per socket:              4
...
+1. Proceed with the [Common](#common) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Wait for the `OPERATING_SYSTEM` to boot.
+1. Run PowerShell as an administrator and check total CPU cores by executing
+command:
+
+```powershell
```
[the remark this answers is not in the document]
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/31T-cpu-status.md]: Done

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Comment by macpijan** [comment]
@sulewskiprzemyslaw Whi is this not merged still, if approved?
I can see we have pre-commit fails.

File: docs/unified-test-documentation/dasharo-compatibility/31T-cpu-status.md [hunk 5]
```diff
@@ -89,6 +113,56 @@ LEVEL4_CACHE_ASSOC                 0
...
+
+1. Proceed with the [Common](#common) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Wait for the `OPERATING_SYSTEM` to boot.
+1. Run PowerShell as an administrator and execute command:
+
+    ```powershell
```
[the remark this answers is not in the document]
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/31T-cpu-status.md:137]: @macpijan

**Pull request merged**